agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -816,7 +814,7 @@ bool monitor_thread::handle_cmdu_vs_message(Socket *sd, ieee1905_1::CmduMessageR
 
         sta_node = mon_db.sta_add(sta_mac, vap_id);
         sta_node->set_ipv4(sta_ipv4);
-        sta_node->set_bridge_4addr_mac(set_bridge_4addr_mac);
+        sta_node->set_bridge_4addr_mac(al_mac);
```

Comment:
Please also remove `bridge_4addr_mac` from `sClientMonitoringParams` and all its use throught the code.